src/rendering/framebufferrenderer.cpp

```diff
@@ -66,955 +66,963 @@ namespace {
-}
-
-void FramebufferRenderer::updateMSAASamplingPattern() {
-    LDEBUG("Updating MSAA Sampling Pattern");
-
-    const int GRIDSIZE = 32;
-    GLfloat step = 2.0f / static_cast<GLfloat>(GRIDSIZE);
-    GLfloat sizeX = -1.0f,
-        sizeY = 1.0f;
-
-    const int NVERTEX = 4 * 6;
-    // openPixelSizeVertexData
-    GLfloat vertexData[GRIDSIZE * GRIDSIZE * NVERTEX];
-
-    for (int y = 0; y < GRIDSIZE; ++y) {
-        for (int x = 0; x < GRIDSIZE; ++x) {
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX] = sizeX;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 1] = sizeY - step;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 2] = 0.0f;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 3] = 1.0f;
-
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 4] = sizeX + step;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 5] = sizeY;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 6] = 0.0f;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 7] = 1.0f;
-
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 8] = sizeX;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 9] = sizeY;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 10] = 0.0f;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 11] = 1.0f;
-
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 12] = sizeX;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 13] = sizeY - step;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 14] = 0.0f;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 15] = 1.0f;
-
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 16] = sizeX + step;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 17] = sizeY - step;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 18] = 0.0f;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 19] = 1.0f;
-
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 20] = sizeX + step;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 21] = sizeY;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 22] = 0.0f;
-            vertexData[y * GRIDSIZE * NVERTEX + x * NVERTEX + 23] = 1.0f;
+        pixelSizeProgram->activate();
+
+        // Draw sub-pixel grid
+        glEnable(GL_SAMPLE_SHADING);
+        glBindVertexArray(pixelSizeQuadVAO);
+        glDisable(GL_DEPTH_TEST);
+        glDepthMask(false);
+        glDrawArrays(GL_TRIANGLES, 0, GRIDSIZE * GRIDSIZE * 6);
+        glBindVertexArray(0);
+        glDepthMask(true);
+        glEnable(GL_DEPTH_TEST);
+        glDisable(GL_SAMPLE_SHADING);
+
+        pixelSizeProgram->deactivate();
+
+        // Now we render the Nx1 quad strip
+        GLuint nOneStripFramebuffer = 0;
+        GLuint nOneStripVAO = 0;
+        GLuint nOneStripVBO = 0;
+        GLuint nOneStripTexture = 0;
+
+        sizeX = -1.0f;
```

Comment:
Overall here:  `1.f` rather than `1.0f`, etc for consistency

Reply:
Ok.

Reply:
See OCD comment above ;)